stages/org.osbuild.machine-id

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python3
+"""
+Deal with /etc/machine-id
+
+Explicitly define the state to /etc/machine-id.
+"""
+
+import pathlib
+import sys
+
+import osbuild.api
+
+SCHEMA = """
+"additionalProperties": false,
+"required": ["machine-id"],
+"properties": {
+  "machine-id": {
+    "enum": ["first-boot", "empty", "preserve"],
+    "description": "Ensure the state of the /etc/machine-id file in the tree"
+  }
+}
+"""
+
+
+def main(tree, options):
+    mode = options["machine-id"]
+
+    # use "match" here once we are on py3.10
```

Comment:
Not really necessary to mention, it will (sadly) be a long time before our minimum supported version is 3.10.

Reply:
Good point, I removed it (and I'm slightly sad)